src/templates/next/components/complex/Infopic/Infopic.tsx

```diff
@@ -122,17 +122,16 @@ const SidePartInfoPic = ({
       {/* Desktop */}
       <div className="hidden lg:block">
         <div
-          className={`${ComponentContent} flex ${
-            isTextOnRight ? "flex-row" : "flex-row-reverse"
-          } gap-10 px-10`}
+          className={`flex ${isTextOnRight ? "flex-row" : "flex-row-reverse"}`}
         >
           <ImageComponent src={src} alt={alt} className="w-1/2" />
-          <div className="w-1/2 py-24 my-auto">
+          <div className="w-1/2 py-24 my-auto pl-10">
```

Comment:
issue: we might need to constrain the height of the component? Otherwise on a wide screen the image might get too tall.

<img width="2560" alt="Screenshot 2024-05-03 at 14 56 12" src="https://github.com/isomerpages/isomer-components/assets/27919917/07d30b55-247c-45ad-b2fe-fcd0cadea6db">

Reply:
Checked with Se Hyun - we're restricting image height to 620px for desktop, and 360px for smaller sizes! Modified in https://github.com/isomerpages/isomer-components/pull/84/commits/e39f6ab544f409bf8296647c7ab43e0ac4e9f575